TestSuites/RDP/RDPSUTControlAgent/Python/RDPSUTControlAgent.py

```diff
@@ -0,0 +1,290 @@
+
+result_code = {'SUCCESS': 0x00000000}
+
+basic_input_flag = {'KEYBOARD_EVENT': 0x00000001,
+                    'UNICODE_KEYBOARD_EVENT': 0x00000002,
+                    'MOUSE_EVENT': 0x00000004,
+                    'EXTENDED_MOUSE_EVENT': 0x00000008,
+                    'CLIENT_SYNCHRONIZE_EVENT': 0x00000010,
+                    'CLIENT_REFRESH_RECT': 0x00000020,
+                    'CLIENT_SUPPRESS_OUTPUT': 0x00000040}
+
+screen_type = {'NORMAL': 0x0000,
+               'FULL_SCREEN': 0x0001}
+
+connect_approach = {'NEGOTIATE': 0x0000,
+                    'DIRECT': 0x0001}
+
+monitor_action = {'ADD_MONITOR': 0x00000001,
+                  'REMOVE_MONITOR': 0x00000002,
+                  'MOVE_MONITOR_POSITION': 0x00000004}
+
+
+class Payload:
+    def __init__(self):
+        self.type = ""
+        self.content = ""
+        self.port = 0
+        self.desktop_width = 0
+        self.desktop_height = 0
+        self.connect_approach = ""
+        self.screen_type = ""
+        self.address = ""
+
+    def encode(self):
+        pass
+
+    def decode(self, raw_payload):
+        self.type = unpack('<i', raw_payload[:4])
+        self.type = self.type[0]
+        print("payload type:", self.type)
+        if self.type == payload_type['RDP_FILE']:
+            content_length = len(raw_payload) - 4
+            self.content = unpack('<%sc' % content_length, raw_payload[4:])
+            print("content", self.content)
+        elif self.type == payload_type['PARAMETERS_STRUCT']:
+            start = 4
+            self.port = unpack('<h', raw_payload[start:start + 2])
+            self.port = self.port[0]
+            print("port", self.port)
+            start += 2
+            self.screen_type = unpack('<h', raw_payload[start:start + 2])
+            self.screen_type = self.screen_type[0]
+            print("screen_type", self.screen_type)
+            start += 2
+            self.desktop_width = unpack('<h', raw_payload[start:start + 2])
+            self.desktop_width = self.desktop_width[0]
+            print("desktop_width", self.desktop_width)
+            start += 2
+            self.desktop_height = unpack('<h', raw_payload[start:start + 2])
+            self.desktop_height = self.desktop_height[0]
+            print("desktop_height", self.desktop_height)
+            start += 2
+            self.connect_approach = unpack('<h', raw_payload[start:start + 2])
+            self.connect_approach = self.connect_approach[0]
+            print("connect_approach", self.connect_approach)
+            start += 2
+            address_length = unpack('<h', raw_payload[start:start + 2])
+            address_length = address_length[0]
+            if address_length:
+                start += 2
+                self.address = unpack(
+                    '<%sc' % address_length, raw_payload[start:start + address_length])
+                self.address = str(b''.join(self.address))
+                print("address", self.address)
+
```

Comment:
If the payloadtype is neither 0 nor 1, the error payloadtype need to be handled.